src/main/java/org/verdictdb/connection/InMemoryAggregate.java

```diff
@@ -156,4 +161,14 @@ private static void resetSchemaAndTableForCombine(UnnamedColumn column) {
     }
   }
 
+  public void abort() {
```

Comment:
Please change the function name "resetSchemaAndTableForCombine" to "resetSchemaAndTableForCombining".